<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="windowModality">
            <enum>Qt::WindowModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>261</width>
                <height>74</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>261</width>
                <height>74</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>261</width>
                <height>74</height>
            </size>
        </property>
        <property name="sizeIncrement">
            <size>
                <width>261</width>
                <height>74</height>
            </size>
        </property>
        <property name="baseSize">
            <size>
                <width>261</width>
                <height>74</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Settings</string>
        </property>
        <property name="windowIcon">
            <iconset>
                <normaloff>logo.png</normaloff>logo.png
            </iconset>
        </property>
        <property name="sizeGripEnabled">
            <bool>false</bool>
        </property>
        <property name="modal">
            <bool>true</bool>
        </property>
        <widget class="QPushButton" name="saveButton">
            <property name="geometry">
                <rect>
                    <x>90</x>
                    <y>40</y>
                    <width>75</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="text">
                <string>Save</string>
            </property>
            <property name="shortcut">
                <string>Return</string>
            </property>
        </widget>
        <widget class="QPushButton" name="cancelButton">
            <property name="geometry">
                <rect>
                    <x>176</x>
                    <y>40</y>
                    <width>75</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="text">
                <string>Cancel</string>
            </property>
            <property name="shortcut">
                <string>Esc</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="txtIP">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>241</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="inputMask">
                <string/>
            </property>
            <property name="placeholderText">
                <string>Enter Your TV IP Address</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
